### PR TITLE
Update euiccs-ddf-file.md

### PR DESCRIPTION
**Pull request by apdutta**
Updating the DDF to reflect the current DDF from the codebase

### CODE REVIEW

**Review by jdbruner — APPROVED** [review]
I believe this is the same content as the DDF file itself, which we've already reviewed and approved.

File: windows/client-management/mdm/euiccs-ddf-file.md [hunk 1]
```diff
@@ -118,6 +118,139 @@ The XML below if for Windows 10, version 1803.
                     </DFType>
                 </DFProperties>
             </Node>
+            <Node>
+                <NodeName>PPR1Allowed</NodeName>
+                <DFProperties>
+                    <AccessType>
+                        <Get />
+                    </AccessType>
+                    <Description>Indicates whether the download of a profile with PPR1 is allowed. If the eUICC has already a profile (regardless of its origin and policy rules associated with it), then the download of a profile with PPR1 is not allowed.</Description>
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/euiccs-ddf-file.md:127]:
```suggestion
                    <Description>Indicates whether the download of a profile with PPR1 is allowed. If the eUICC already has a profile (regardless of its origin and policy rules associated with it), the download of a profile with PPR1 is not allowed.</Description>
```

File: windows/client-management/mdm/euiccs-ddf-file.md [hunk 2]
```diff
@@ -118,6 +118,139 @@ The XML below if for Windows 10, version 1803.
...
+                    </DFType>
+                </DFProperties>
+            </Node>
+            <Node>
+                <NodeName>PPR1AlreadySet</NodeName>
+                <DFProperties>
+                    <AccessType>
+                        <Get />
+                    </AccessType>
+                    <Description>Indicates whether the eUICC has already a profile with PPR1.</Description>
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/euiccs-ddf-file.md:148]:
```suggestion
                    <Description>Indicates whether the eUICC already has a profile with PPR1.</Description>
```

File: windows/client-management/mdm/euiccs-ddf-file.md [hunk 3]
```diff
@@ -118,6 +118,139 @@ The XML below if for Windows 10, version 1803.
...
+                <Node>
+                    <NodeName></NodeName>
+                    <DFProperties>
+                        <AccessType>
+                            <Add />
+                            <Delete />
+                            <Get />
+                            <Replace />
+                        </AccessType>
+                        <Description>Node representing the discovery operation for a server name. The node name is the fully qualified domain name of the SM-DP+ server that will be used for profile discovery. Creation of this subtree triggers a discovery request.</Description>
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/euiccs-ddf-file.md:192]:
```suggestion
                        <Description>Node specifying the server name for a discovery operation. The node name is the fully qualified domain name of the SM-DP+ server that will be used for profile discovery. Creation of this subtree triggers a discovery request.</Description>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by illfated — APPROVED** [review]
Regular source file view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9137/files

Rich Diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9137/files?short_path=ba9dc2e#diff-ba9dc2e81715b572193445278a2b7ceb5963e41538b155edc51ecb818379e088

Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**